Sources/DesignSystem/Views/BraveTextFieldStyle.swift

```diff
@@ -83,12 +83,14 @@ public struct BraveValidatedTextFieldStyle<Failure: LocalizedError & Equatable>:
   }
 }
 
-private struct BaseBraveTextFieldStyleModifier: ViewModifier {
+struct BaseBraveTextFieldStyleModifier: ViewModifier {
```

Comment:
Rename to `BraveTextInputStyleModifier` since you're using it for both text fields + text views

Reply:
Done